backend/src/waterfallhunter/core/notifier.py

```
@@ -70,60 +387,110 @@ def build_signal_message(cls, symbol: str, data: dict) -> str:
-                        for update in updates:
-                            self.offset = update["update_id"] + 1
-                            message = update.get("message", {})
-                            chat = message.get("chat", {})
-                            text = message.get("text", "")
-
-                            if str(chat.get("id")) == self.chat_id and text.startswith("/"):
-                                await self._process_command(text)
+                        if updates:
+                            self.offset = updates[-1]["update_id"] + 1
             except Exception:
                 pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)